packages/sku/lib/validateLessFiles.js

```diff
@@ -0,0 +1,38 @@
+const fs = require('fs');
+const path = require('path');
+const glob = require('fast-glob');
+const chalk = require('chalk');
+
+const printBanner = require('./banner');
+const exists = require('./exists');
+const { paths } = require('../context');
+
+module.exports = async () => {
+  const srcPathsExist = await Promise.all(
+    paths.src.map(async (srcPath) => (await exists(srcPath)) && srcPath),
```

Comment:
Would `filter` work here?
```suggestion
    paths.src.filter(async (srcPath) => await exists(srcPath)),
```